pyproject.toml

```
@@ -4,7 +4,7 @@ target-version = 'py311'
 
 [tool.poetry]
 name = "redbox"
-version = "1.0.0"
+version = "0.0.3"
```

Aren't we on `0.3.0`?

```suggestion
version = "0.3.0"
```